titiler/api/endpoints/stac.py

```diff
@@ -150,10 +150,7 @@ async def stac_tile(
     headers: Dict[str, str] = {}
 
     if not image_params.expression and not assets:
-        raise HTTPException(
-            status_code=status.HTTP_403_FORBIDDEN,
-            detail="Must pass Expression or Asset list.",
-        )
+        raise BadRequestError("Must pass Expression or Asset list.")
```

Comment:
maybe we should let rio-tiler raise here https://github.com/cogeotiff/rio-tiler/blob/master/rio_tiler/io/stac.py#L227 (this is in rio-tiler 2.0b2)

Reply:
@geospatial-jeff I fixed the exception issue in rio-tiler-crs, you can pin rio-tiler-crs~=3.0b2 https://pypi.org/project/rio-tiler-crs/3.0b2/